package.json

```diff
@@ -54,6 +54,7 @@
     "react-redux": "4.4.5",
     "react-router": "2.7.0",
     "react-router-redux": "3.0.0",
+    "react-window-resize-listener": "1.1.0",
```

Comment:
I agree.  This repo looks pretty stale.  I've changed over to the suggested forked repo.